### PR TITLE
CORE-7512 display requester email and request status history under Up…

### PR DESCRIPTION
**Pull request by sri2k1us**
…date request Dialog in Belphegor.

### CODE REVIEW

File: ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/UpdatePermanentIdRequest.ui.xml [hunk 1]
```diff
@@ -19,9 +20,16 @@
 
     <container:VerticalLayoutContainer>
         <container:child layoutData="{layoutData}">
+            <form:FieldLabel text="Email">
```
Comment (ash3rz) [inline ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/UpdatePermanentIdRequest.ui.xml]: This is minor but you could probably move the "Email" text to the appearance, like you have in the next FieldLabel for consistency.

File: ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/presenter/PermanentIdRequestPresenter.java [hunk 2]
```diff
@@ -131,6 +136,41 @@ public void onSuccess(String result) {
...
+                view.unmask();
+                IplantErrorDialog ied = new IplantErrorDialog(I18N.DISPLAY.error(), caught.getMessage());
+                ied.show();
+            }
+
+            @Override
+            public void onSuccess(String result) {
+                view.unmask();
+                final AutoBean<PermanentIdRequesDetails> decode =
+                        AutoBeanCodex.decode(factory, PermanentIdRequesDetails.class, result);
```
Comment (psarando) [inline ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/presenter/PermanentIdRequestPresenter.java]: This AutoBean decoding could be done in the service facade with an `AsyncCallbackConverter`.

File: ui/de-lib/src/main/java/org/iplantc/de/client/models/identifiers/PermanentIdRequesDetails.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+import java.util.List;
+
+/**
+ * 
+ * 
+ * @author sriram
+ * 
+ */
+public interface PermanentIdRequesDetails {
```
Comment (ash3rz) [inline ui/de-lib/src/main/java/org/iplantc/de/client/models/identifiers/PermanentIdRequesDetails.java]: Also minor, but "Request" is missing the T.

File: ui/de-lib/src/main/java/org/iplantc/de/client/models/identifiers/PermanentIdRequestAutoBeanFactory.java [hunk 4]
```diff
@@ -16,4 +16,6 @@
     AutoBean<PermanentIdRequestList> getAllRequests();
 
     AutoBean<PermanentIdRequestUpdate> getStatus();
+
+    AutoBean<PermanentIdRequesDetails> getDeatils();
```
Comment (ash3rz) [inline ui/de-lib/src/main/java/org/iplantc/de/client/models/identifiers/PermanentIdRequestAutoBeanFactory.java]: "Details" is spelled incorrectly :stuck_out_tongue:

**Pull request merged**